core/src/main/java/org/lflang/generator/EnclaveInfo.java

```diff
@@ -0,0 +1,18 @@
+package org.lflang.generator;
+
+public class EnclaveInfo {
```

Comment:
Would it be better to use a `record` here (instead of a `class`)?

Reply:
AFAIK the fields of a record are immutable, I would like to change the fields of the enclaveInfo as we explore the reactor-graph